public/custom/emberjs/default.html

```diff
@@ -18,9 +20,9 @@
 
   <script type="text/x-handlebars" data-template-name="index">
     <ul>
-    {{#each item in model}}
-      <li>{{item}}</li>
-    {{/each}}
+      {{#each color in model}}
```

Comment:
Can you update this to the latest recommended syntax `{{#each model as |color|}}`